projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -747,6 +748,19 @@ public static function is_enabled_jetpack_recurring_payments() {
 		return $api_available;
 	}
 
+	/**
+	 * Whether to enable the blocks in the editor.
+	 * All Monetize blocks (except Simple Payments) need an active connecting and a user with at least `edit_posts` capability
+	 *
+	 * @return bool
+	 */
+	public static function should_enable_monetize_blocks_in_editor() {
+		$jetpack_ready_and_user_connected = Jetpack::is_connection_ready() &&
+			( new Connection_Manager( 'jetpack' ) )->is_user_connected() && current_user_can( 'edit_posts' );
```

Comment:
I don't know if we need the `edit_posts` cap check here? Since the block editor already limits access to folks with `edit_posts` capabilities?

Reply:
I agree. My question however was: is the current user the same as the WPCOM one? I am struggling to understand the connection between both, and if
```
new Connection_Manager( 'jetpack' ) )->is_user_connected()
```
Actually means the user is connected to WPCOM?

Reply:
My IDE and I (!!) cannot find the implementation for the `Automattic\Jetpack\Connection\Manager` 😅

Reply:
That's interesting. I take it it eventually worked for you and you found the file?